viewing_party/party.py

```diff
@@ -1,23 +1,123 @@
 # ------------- WAVE 1 --------------------
 
+from hashlib import new
+from re import M
+from collections import Counter
+from viewing_party.helper_functions import *
+
+from tests.test_constants import GENRE_1, MOVIE_TITLE_1, RATING_1, USER_DATA_3
+
 def create_movie(title, genre, rating):
-    pass
+    if title and genre and rating:
+        new_movie = {
+            "title": title,
+            "genre": genre,
+            "rating": rating
+        }
+        return new_movie
+    else:
+        return None
+
+def add_to_watched(user_data, movie):
+    user_data = {
+        "watched": [movie]
+    }             

```

Comment:
With this implementation, if there were already any movies in a user's watched list then you'd completely overwrite the existing list by setting the key "watched" to the new list that comes from `movie`. 

To add to the existing list without overwriting what's already in "watched", we'd use the append method.